ui/src/main/webapp/src/app/registered-application/application-queue-list.component.html

```diff
@@ -6,14 +6,13 @@
                  aria-valuemin="0" aria-valuemax="100" aria-valuenow="100"
                  style="width: 100%">
                 <div class="file-info">
-                    <span>{{application.title}}</span>
+                    <span>{{application.title}} 100%</span>
```

Comment:
I did it because in the case of a narrow page and a long file name, they'll overlap and they'll be not readable anymore. See first file entry in the next screenshot.
![screenshot from 2017-06-13 19-22-42](https://user-images.githubusercontent.com/7288588/27095468-0554b210-506e-11e7-9c90-98bd14927cc6.png)

To put the percentage after the file name (and, in the near future, after the file size) is only functional to avoid weird label overlaps.